code/ui/Scoreboard.cs

```diff
@@ -0,0 +1,175 @@
+using Sandbox;
+using Sandbox.UI;
+using Sandbox.UI.Construct;
+using System.Collections.Generic;
+
+namespace TTTReborn.UI
+{
+    public class Scoreboard : Panel
+    {
+        public Scoreboard()
+        {
+            StyleSheet.Load("/ui/Scoreboard.scss");
+            header = new Header();
```

Comment:
This can be created inside of `AddHeader()`?

Reply:
@Westsaid

Reply:
Not sure if you want to test / address since westsaid is gone for the week. You should be able to simply move `header` into `AddHeader`